src/main.c

```diff
@@ -784,6 +787,16 @@ static NETX_CONSOLEAPP_RESULT_T opMode_identify(void)
 }
 
 
+/* ------------------------------------- */
+static NETX_CONSOLEAPP_RESULT_T opMode_reset(void)
+{
+	NETX_CONSOLEAPP_RESULT_T retVal = NETX_CONSOLEAPP_RESULT_ERROR;
+	uprintf("Activating watchdog\n");
+	retVal = resetNetX();
+	uprintf("Watchdog activated\n");
```

Comment:
This might cause issues. 
The connection must be closed before the reset occurs. 
The uprintf after the watchdog has been activated may lead to an unwanted delay.